week6_outro/practice_mcts.ipynb

```diff
@@ -203,13 +213,30 @@
+    }
+   },
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "# MCTS: Monte-Carlo tree search\n",
     "\n",
+    "![MCTS.png](attachment:MCTS.png)\n",
```

Comment:
Let's choose one of:
1) embed images to notebooks
2) host images in our repo
3) host images on some image hosting
4) find all the images on the existing sources.
Currently we use all 4 ways in different places.

Reply:
Out of scope for this PR, but certainly worth creating an issue